### ISSUE TITLE
[MOBILE] headless cms - code bracket image

### ISSUE THREAD

**Issue by Katiemoser** [zesty-io/nextjs-website#205]
Please drop size of this image down by 25% and move above text in this section

![Screenshot_20220321-092054_Chrome.jpg](https://user-images.githubusercontent.com/101665566/159328673-6e9e9733-6613-4bf0-ae52-b35ca556bc73.jpg)

**Comment by darwin808** [comment]
![Screenshot_2022-03-22_02-39-44](https://user-images.githubusercontent.com/71545960/159341816-ac2b1f7e-646c-44ca-a2ea-1e0ed3c02696.png)
![Screenshot_2022-03-22_02-40-18](https://user-images.githubusercontent.com/71545960/159341822-988a91c7-9046-4bbb-bab4-582c07f1218e.png)

**Comment by Katiemoser** [comment]
Can we make the image show up smaller? like maybe 75% of that size?

**Comment by darwin808** [comment]
![Screenshot_2022-03-22_03-02-31](https://user-images.githubusercontent.com/71545960/159345326-60e46883-9332-40c4-800c-29105510f4c6.png)
![Screenshot_2022-03-22_03-02-38](https://user-images.githubusercontent.com/71545960/159345350-0bd18afa-5ea6-4970-a4f5-c5f9b71595b6.png)

**Comment by darwin808** [comment]
like this?

**Comment by Katiemoser** [comment]
Yes, looks great on desktop. On mobile, can we decrease the padding a bit between the header above and the image?

**Comment by darwin808** [comment]
![Screenshot_2022-03-22_04-08-46](https://user-images.githubusercontent.com/71545960/159355428-ed5c6076-82a2-4bce-85cd-1eeec7adffc3.png)
![Screenshot_2022-03-22_04-08-59](https://user-images.githubusercontent.com/71545960/159355435-ab279b5b-fbda-47db-836c-b50a10b52744.png)

**Comment by Katiemoser** [comment]
Approved!
